crowbar_framework/app/assets/stylesheets/content/content.scss

```diff
@@ -43,6 +33,47 @@
       margin-bottom: 0;
     }
   }
+
+  .panel-plain {
+    @include clearfix;
+  }
+
+  .panel-footer {
+    .progress {
+      background-color: lighten(#f5f5f5, 50%);
```

Comment:
Color literals like `#f5f5f5` should only be used in variable declarations; they should be referred to via variable everywhere else.